src/xdocs/checks/design/onetoplevelclass.xml.template

```diff
@@ -20,58 +20,42 @@
       </subsection>
 
       <subsection name="Examples" id="Examples">
-        <p>
+        <p id="Example1-config">
           To configure the check:
         </p>
-        <source>
-&lt;module name="OneTopLevelClass"/&gt;
-        </source>
+        <macro name="example">
+          <param name="path"
+                 value="resources/com/puppycrawl/tools/checkstyle/checks/design/onetoplevelclass/Example1.txt"/>
+          <param name="type" value="config"/>
+        </macro>
         <p>
           <b>ATTENTION:</b> This Check does not support customization of validated tokens, so do
           not use the "tokens" property.
         </p>
-        <p>
+        <p id="Example1-code">
           An example of code with violations:
         </p>
-        <div class="wrapper">
-          <pre>
-public class Foo { // OK, first top-level class
-  // methods
-}
-
-class Foo2 { // violation, second top-level class
-  // methods
-}
-
-record Foo3 { // violation, third top-level "class"
-  // methods
-}
-          </pre>
-        </div>
-        <p>
+        <macro name="example">
+          <param name="path"
+                 value="resources/com/puppycrawl/tools/checkstyle/checks/design/onetoplevelclass/Example1.txt"/>
+          <param name="type" value="code"/>
+        </macro>
+        <p id="Example2-code">
           An example of code without public top-level type:
         </p>
-        <div class="wrapper">
```

Comment:
These `wrapper` divs are only used for tables as far as I am aware. I assume this was the old way of showing code in the website and some examples didn't get converted to `<source>`

Reply:
I think using `wrapper` has a bad side effect too, compare these two examples.

From the currently published website (https://checkstyle.org/checks/design/onetoplevelclass.html) :
![image](https://github.com/checkstyle/checkstyle/assets/31252532/52a97387-5027-46dd-97be-90ed22e9162f)

From the site generated in this PR (https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/a63cc64_2023095803/checks/design/onetoplevelclass.html#Examples) :
![image](https://github.com/checkstyle/checkstyle/assets/31252532/6166ff5d-8df3-4cf6-9960-1b54cd092484)

It seems like wrapper does not enclose the code examples in the "frame" like `source` does. @stoyanK7 thanks for finding this.

Reply:
@romani ^